src/devices/bus/qbus/dvk_mx.h

```diff
@@ -0,0 +1,172 @@
+// license:BSD-3-Clause
+// copyright-holders:Sergey Svishchev
+/***************************************************************************
+
+	DVK "MX" floppy controller (decimal ID 3.057.122)
+
+***************************************************************************/
+
+#pragma once
+
+#ifndef __DVK_MX_H__
+#define __DVK_MX_H__
```

Comment:
Please make the `#include` guard macro name match the path – it should be `MAME_BUS_QBUS_DVK_MX` for this header.  Also, the `#pragma once` should go inside the `#ifndef`.  Look at other headers for examples.